File: MCP_Backend-master/src/main/java/com/greybox/mediums/models/CommissionTierData.java
package com.greybox.mediums.models;

import com.greybox.mediums.entities.ServiceCommissionTier;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class CommissionTierData {
    private ServiceCommissionTier[] data;
    private Integer commissionId;
    private String createdBy;
    private Date createDate;
}

File: MCP_Frontend-main/src/app/shared/models/commission-tier-data.ts
export class CommissionTier {
    commissionTierId: number;
    commissionId: number;
    tierNo: number;
    fromAmt: number;
    toAmt: number;
    commissionAmount: number;
    createDate: Date;
    createdBy: string;
    modifyDate: Date;
    modifiedBy: string;
}

export class CommissionTierData {
    data: CommissionTier[];
    commissionId: number;
    createdBy: String;
    createDate: Date;
}